tutorial/index.jade

```diff
@@ -152,25 +155,25 @@
       ## Verifying the Code
 
       Authy provides us with a `tokens.verify` method that allows us to pass a
-      user id, and token. In this case we just
-      need to check that the API request was successful and if so set
-      a flag on the user that they were verified. 
+      user id and token. In this case we just
+      need to check that the API request was successful and if so, set
+      a flag on the user that they were verified.
 
       That's all for token verification!
 
   .step(data-file='account_verification_flask/services/twilio_services.py')
     :markdown
       ## Sending the Confirmation Message
 
-      Much as we did for our Authy client, we create a single instance of the
-      Twilio REST API helper, called `twilio_client` in this example.
+      Just as we did for our Authy client, we create a single instance of the
+      Twilio REST API helper. It will be called `twilio_client` in this example.
 
-      Then all we need to do to send an sms to the user's phone is use the built
+      Then all we need to do is send an SMS to the user's phone that was used
```

Comment:
"that was" makes this sentence clumsy. "Then all we need to do is send an SMS to the phone number used in "messages.create()" is much better.